File: python/01_estrutura_sequencial/ex04.py
"""
Exercício 04

Faça um Programa que peça as 4 notas bimestrais e mostre a média.
"""

notas = []
for i in range(4):
    nota = int(input("Digite uma nota: "))
    notas.append(nota)
print(f"A média das notas é: {sum(notas) / len(notas)}")

File: nodejs/01-estrutura-sequencial/ex04.js
/*
 * Exercício 04
 *
 * Faça um Programa que peça as 4 notas bimestrais e mostre a média.
 */

import * as readlinePromises from 'node:readline/promises'
import {stdin as input, stdout as output} from 'node:process'

const rlp = readlinePromises.createInterface({input, output})
const notas = new Array(4)
for (let i = 0; i < notas.length; i++) {
    const nota = await rlp.question("Digite uma nota: ")
    notas[i] = parseFloat(nota)
}
rlp.close()
const media = notas.reduce((a, b) => a + b, 0) / notas.length
console.log(`A média das notas é: ${media}`)
